utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -597,17 +587,16 @@ void PessimisticTransactionDB::GetAllPreparedTransactions(
   }
 }
 
-TransactionLockMgr::LockStatusData
-PessimisticTransactionDB::GetLockStatusData() {
-  return lock_mgr_.GetLockStatusData();
+LockManager::PointLockStatus PessimisticTransactionDB::GetLockStatusData() {
+  return lock_manager_->GetPointLockStatus();
```

Comment:
So, now LockManager got two functions, `GetPointLockStatus` and `GetRangeLockStatus`. 

Should this function (`PessimisticTransactionDB::GetLockStatusData`) also be replaced with two functions? (And if yes, should this be done in this PR or outside of it?)

Reply:
I'll add these two methods and add a comment in `TransactionDB` to mention that `GetLockStatusData` is deprecated in favor of `GetPointLockStatus`.

Reply:
I talked with @cheng-chang  about this, and I think that in the future, we should always get status from the lock manager objects, and the methods on `PessimisticTransactionDB` will go away.

I think the below calls relating to deadlock info would be similar too.